src/components/Quote.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Quote = () => (
+  <div style={{
+    width: '60%',
+    margin: '4% auto',
+    fontSize: '1.1rem',
+  }}
```

Comment:
- [ ] It is not good practice to have your styling right here in the JSX. You can create a Quote.css file or simply write the styling in the App.css file instead 👍